src/dfx/src/lib/operations/canister/deploy_canisters.rs

```diff
@@ -325,7 +329,7 @@ async fn install_canisters(
     no_asset_upgrade: bool,
     always_assist: bool,
 ) -> DfxResult {
-    info!(env.get_logger(), "Installing canisters...");
+    let spinner = env.new_spinner("Installing canisters...".into());
```

Comment:
This also conflicts with prompting about upgrades or for initialization arguments:
```
$ dfx deploy
...
⠂ Installing canisters...
⠁ Installing code for canister dd_backend, with canister ID br5f7-7uaaa-aaaaa-qaaca-cai                                                              
This canister requires an initialization argument.
Auto-completions: aaabbbccc, abc3, abc4, alice, alice_e2e, anonymous, bob, bob_e2e, dd_backend, dd_frontend, default, ed, iii, k3, kt2, kt4, mainnet,⠠ Installing canisters...
⠤ Installing code for canister dd_backend, with canister ID br5f7-7uaaa-aaaaa-qaaca-cai  
```